xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEStream.h

```diff
@@ -179,6 +180,7 @@ class CActiveAEStream : public IAEStream
 
 protected:
 
+  CActiveAE *m_activeAE;
```

Comment:
use a reference to CActiveAE to make it explicit to the compiler and reader that it's valid for the lifetime of this class.